README.md

```diff
@@ -18,20 +20,48 @@ Install Node.js dependencies:
 npm install
 ```
 
+### Configuration
+
+There are two ways of passing application options for PostgreSQL and OAuth:
+
+1) Using environment variables, full list available at [config/custom-environment-variables.json](config/custom-environment-variables.json)
+
+2) Adding file `config/local-development.json`. It will not be git-tracked and will override options for `development` environment. 
```

Comment:
It maybe useful to have a `config/local-development.json.example` or so checked in to the repository. It was not completely obvious how this file should look. Mine ended up being like this:

```
{
  "osmOAuth": {
    "requestTokenUrl": "https://master.apis.dev.openstreetmap.org/oauth/request_token",
    "accessTokenUrl": "https://master.apis.dev.openstreetmap.org/oauth/access_token",
    "authorizeUrl": "https://master.apis.dev.openstreetmap.org/oauth/authorize",
    "profileUrl": "https://master.apis.dev.openstreetmap.org/api/0.6/user/details",
    "clientId": "xxx",
    "clientSecret": "xxx"
  }
}
```